src/components/vehicleDataGroup/VehicleDataGroupField.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactNode } from 'react';
+import { Box, chakra, ListItem, useMultiStyleConfig } from '@chakra-ui/react';
+
+interface VehicleGroupDataFieldProps {
+  label: ReactNode;
+  value: ReactNode;
+}
+
+const VehicleGroupDataField: FC<VehicleGroupDataFieldProps> = ({
+  label,
+  value,
+}) => {
+  const { listItem, listItemElement } = useMultiStyleConfig(`VehicleDataGroup`);
+
+  return (
+    <ListItem>
+      <Box __css={listItem}>
+        <chakra.span __css={listItemElement}>{label}</chakra.span>
```

Comment:
Could we use the `Text` component so we don't have to redefine the text styles?

Reply:
> Could we use the `Text` component so we don't have to redefine the text styles?

Hm, I tried to apply __css prop to Text comp but it does not read __css prop. The same situation I had with ListItem comp

Reply:
You can use the `Text` component and use its props for changing its style. For example: `color`, `textStyle` etc.